plugins/source/snyk/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/snyk/resources/plugin"
+	"github.com/cloudquery/plugin-sdk/serve"
+)
+
+const sentryDSN = "TODO"
```

Comment:
This is left as `TODO`, did you force push without pulling?

Reply:
ooh

Reply:
fixed